react-components/src/components/Widgets/WidgetBody.tsx

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+import { type PropsWithChildren, type ReactNode, type ReactElement } from 'react';
+import styled from 'styled-components';
+
+export const WidgetBody = ({ children }: PropsWithChildren): ReactElement => {
+  const renderContent = (): ReactNode => {
+    return children;
+  };
+
+  return <Container>{renderContent()}</Container>;
```

Comment:
Can we just write `{children}` here? I wonder if it's more clear to just create a styled container in the `Widget.tsx` instead of having this declared as a component in a separate file